Jenkinsfile

```diff
@@ -70,15 +70,23 @@ spec:
         stage('Bootstrap') {
             echo sh(script: 'env|sort', returnStdout: true)
         }
-        stage('Install a Maven project') {
+        stage('Unit Test, Checkstyle and Install') {
             git branch: "${env.BRANCH_NAME}", url: 'https://github.com/gchq/Palisade-services.git'
             container('docker-cmds') {
                 configFileProvider([configFile(fileId: "${env.CONFIG_FILE}", variable: 'MAVEN_SETTINGS')]) {
                     sh 'mvn -s $MAVEN_SETTINGS install'
                 }
             }
         }
-        stage('Deploy a Maven project') {
+        stage('Integration Tests') {
+            git branch: "develop", url: 'https://github.com/gchq/Palisade-integration-tests.git'
```

Comment:
This is a Jenkins build for this repository only. The integration-test branch itself will need a Jenkins build in it's own right.